.circleci/config.yml

```diff
@@ -40,20 +38,13 @@ jobs:
           key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
       - run:
           command: |
-              python3 -m venv venv
-              . venv/bin/activate
-              pip install -r requirements.txt
+              make backend-deps
       - save_cache:
             key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
             paths:
               - "venv"
       - run:
-          command: |
-              . venv/bin/activate
-              DJANGO_SETTINGS_MODULE=kamu.settings.test coverage run manage.py test
-              coverage xml
-              ./tmp/cc-test-reporter format-coverage -t coverage.py -o tmp/codeclimate.backend.json coverage.xml
-      - persist_to_workspace:
```

Comment:
Was the `persist_to_workspace` really supposed to be removed?

Reply:
I think not, let me add again. I need to stop using drugs. /o\